packages/atomic/src/components/random-customer-component/random-customer-component.tsx

```diff
@@ -0,0 +1,23 @@
+import {Engine, HeadlessEngine, searchAppReducers} from '@coveo/headless';
+import {Component, h} from '@stencil/core';
+
+@Component({
+  tag: 'random-customer-component',
+  shadow: true,
+})
+export class RandomCustomerComponent {
+  engine: Engine;
+
+  constructor() {
+    this.engine = new HeadlessEngine({
+      configuration: HeadlessEngine.getSampleConfiguration(),
+      reducers: searchAppReducers,
+    });
+  }
+  render() {
+    return [
+      <atomic-search-box engine={this.engine}></atomic-search-box>,
+      <atomic-result-list engine={this.engine}></atomic-result-list>,
+    ];
```

Comment:
Hmm, similar to the other PRs, I think it would be simpler to place the custom component inside `atomic-search-interface`, not passing the engine to each component. e.g.

```
<atomic-search-interface>
    <random-customer-component/>
</atomic-search-interface>
```

and if someone is building component that needs access to the engine, can't we just export the `initialization` decorator that will get the correct engine instance and set it on the component?